classes/pdf/PDFGenerator.php

```diff
@@ -147,7 +148,7 @@ public function setFontForLang($iso_lang)
         if (array_key_exists($iso_lang, $this->font_by_lang)) {
             $this->font = $this->font_by_lang[$iso_lang];
         }else {
-            $this->font = self::DEFAULT_FONT;
+            $this->font = 'dejavusans';
```

Comment:
You could have modified the value `DEFAULT_FONT` on line 35, because this is the actual purpose of this variable to set a fallback value.

But while the other change can be considered as a bug fix, this one may change the whole behavior of the class for all undefined languages. This needs tests prior any approval.